### DISCUSSION TITLE
Code

### DISCUSSION THREAD

**Discussion by Irontony08** [community/community#135550] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How to use flex

**Comment by LavishT04** [comment]
Before the Flexbox Layout module, there were four layout modes:

    Block, for sections in a webpage
    Inline, for text
    Table, for two-dimensional table data
    Positioned, for explicit position of an element

The Flexible Box Layout Module, makes it easier to design flexible responsive layout structure without using float or positioning.


 <div class="flex-container">
  <div>1</div>
  <div>2</div>
  <div>3</div>
</div>

**Comment by LiteBrite82** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
